site/docs/numbers/porting/csrLookupTool.mdx

```diff
@@ -184,19 +152,10 @@ Content-Type: application/xml; charset=utf-8
 
 A <Highlight color="#9a59c5">PUT</Highlight> method provides the ability to replace the existing csr conditional data with specificed csr conditional data. To cancel CSR order, users must provide a `Status` element with `REQUESTEDCANCEL` value. Note that csr order must be a non-real time with `PROCESSING` or `ACTIONREQUIRED` status.
```

Comment:
Change this to 

A <Highlight color="#9a59c5">PUT</Highlight> method provides the ability to update a CSR order with additional information requested by the carrier. CSR orders will be in either a `PROCESSING` or `ACTION_REQUIRED` status in this scenario.
The PUT method can also be used to cancel a CSR order. You must submit a request body with a `Status` element set to `REQUESTED_CANCEL` in order for the cancellation process to begin.